src/app/modules/onboarding/module.nim

```diff
@@ -35,6 +37,7 @@ type
     viewVariant: QVariant
     controller: Controller
     localPairingStatus: LocalPairingStatus
+    currentFlow: SecondaryFlow
```

Comment:
I think you can rename it here in NIM to sth like `OnboardingFlow`; I'll do the same in enums.h but I think it'd be nice to actually use the one from enums.h